usagov/vote-gov-nvrf-app
# VOTE-2720 Update to the latest NVRF PDF (English)

Updated to the latest NVRF PDF (2024 version). 
Remediated the back end PDF data.
Changed pathing to use absolute paths and language codes in the PDF name.

To test:
1. Build and open up the local version of the form.
2. Select "Update Regristration"
3. Select the checkbox for previous name. Fill out all personal info fields.
4. Select the mail checkbox and move checkbox. Fill out all address fields.
5. Continue filling out the rest of the fields.
6. Confirm all data shows on the confirmation page.
7. Open up the PDF in the browser and through download button. Confirm all fields in teh PDF are filled out with the correct data.
